Sources/Control/Extensions/MPVolumeView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  MPVolumeView.swift
+//  ControlKit
+//
+
+import MediaPlayer
+
+extension MPVolumeView {
+    
+    static var volume: Float {
+        get {
+            shared.slider?.value ?? 0
+        }
+        set {
+            shared.slider?.value = newValue
+        }
+    }
+    
+    static func increaseVolume(_ amount: Float) {
+        guard volume < 1 else {
+            print("🚨 Volume is already at max")
+            return
+        }
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.01) {
+            volume += amount
+        }
+    }
+    
+    static func decreaseVolume(_ amount: Float) {
+        guard volume > 0 else {
+            print("🚨 Volume is already at min")
+            return
+        }
+        DispatchQueue.main.asyncAfter(deadline: .now() + 0.01) {
```

Comment:
Out of curiosity 🤓,  why do we need a small delay here?

Reply:
This _may_ have been copy-pasta'd from an LLM 🤖🍝

Monkey see 🙈 monkey do 🙊